src/test/parse-fail/mod_file_not_exist.rs

```diff
@@ -8,6 +8,8 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// ignore-windows
```

Comment:
It doesn't seem OK to ignore this test on Windows.

@alexcrichton What should we do here? This PR is changing a few places to use the native path separator when printing file paths, so I assume this test is failing because the error messages don't compare equal. Is there a way to do conditional testing based on platform?